demos/python_demos/rag_chatbot/README.md

```diff
@@ -0,0 +1,163 @@
+# LLM-powered RAG chatbot serving via Python Calculator in MediaPipe Graph {#ovms_demo_python_rag_chatbot}
```

Comment:
I wonder if saying `via Python Calculator` is relevant here. It's an implementation detail that user might not really care about - the important part is what it enables - at least in the title in my opinion. I'd change it to something like `using Python libraries`. What do you think?

Reply:
Done